aggregator/options.go

```diff
@@ -116,10 +125,10 @@ type options struct {
 	quantilesPool             pool.FloatsPool
 
 	// Derived options.
-	fullCounterPrefix     []byte
-	fullTimerPrefix       []byte
-	fullGaugePrefix       []byte
-	timerQuantileSuffixes [][]byte
+	fullCounterPrefix []byte
+	fullTimerPrefix   []byte
+	fullGaugePrefix   []byte
+	suffixMap         map[policy.AggregationType][]byte
```

Comment:
As discussed offline, could preallocated a `[][]byte` up to the max id and have an `RawID` function that indexes into the `[][]byte` to get the raw id back.